packages/create-docusaurus/templates/classic/src/css/custom.css

```diff
@@ -6,16 +6,27 @@
 
 /* You can override the default Infima variables here. */
 :root {
-  --ifm-color-primary: #25c2a0;
-  --ifm-color-primary-dark: rgb(33, 175, 144);
-  --ifm-color-primary-darker: rgb(31, 165, 136);
-  --ifm-color-primary-darkest: rgb(26, 136, 112);
-  --ifm-color-primary-light: rgb(70, 203, 174);
-  --ifm-color-primary-lighter: rgb(102, 212, 189);
-  --ifm-color-primary-lightest: rgb(146, 224, 208);
+  --ifm-color-primary: #18816a;
```

Comment:
couldn't we use something based on the hsl technique instead of these hex values?

I'd like to encourage users to do so in the future, and eventually have a single color option in our theme to compute the palette automatically, that would be a good first step

Reply:
I'm not sure. The current code is simpler for non-designer users, and in fact the other shades seem to be never used in Infima itself anyways... I'll see how to make that happen